### PR TITLE
Add document for inline attribute

### PR DESCRIPTION
**Pull request by hoantran-it**
Firstly, I want to say thank you for this library! It's very helpful!
When I want to scroll selected tags only, I dive deeply inside source code and find out that we already support it, just missing some document. I create this PR to contribute a bit for project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -365,6 +366,34 @@ The maxLength attribute added to the input. Specifies the maximum number of char
     ...>
 ```
 
+<a name="inline"></a>
+##### inline (optional)
+The inline attribute use to determine that selected tags and input field will be put in the same div block or not. If you want to separate it and apply auto scroll for just selected tags only, change this attribute. Specifies the inline attribute.
```
Comment (ad1992) [inline README.md]: The inline attribute is deciding whether to render the  input field in same div as selected tags or the sibling div. So I think we can remove this line
`If you want to separate it and apply auto scroll for just selected tags only, change this attribute. Specifies the inline attribute
`

Comment (ad1992) [inline README.md]: Can you change the text to  `The inline attributes decides whether the input fields and selected tags will be rendered in-line` ?

File: README.md [hunk 2]
```diff
@@ -365,6 +366,34 @@ The maxLength attribute added to the input. Specifies the maximum number of char
...
+    ...>
+```
+
+If inline is true (default value)
+
+```
+<div className="...">
+	{tagItems}
+	{tagInput}
+</div>
```
Comment (ad1992) [inline README.md]: I would recommend adding screen shots in both cases instead of showing how its rendered in div

Comment (hoantran-it) [reply README.md]: Hi @ad1992, I fix them all as request. Please help to take a look. Thanks!

Comment (ad1992) [reply README.md]: Thanks @hoantran-it  :) .The screenshots look too big and in both the case it seems that they are not inline. Can you decrease the size of screenshots may be something like this 
screen shot 2018-03-17 at 5 20 12 am
screen shot 2018-03-17 at 5 21 34 am

File: README.md [hunk 3]
```diff
@@ -365,6 +366,38 @@ The maxLength attribute added to the input. Specifies the maximum number of char
...
+<ReactTags
+    inline = {false}
+    ...>
+```
+
+If inline is true (default value)
+
+```
+<div className="...">
+	{tagItems}
```
Comment (ad1992) [inline README.md]: This won't be needed after adding the screenshot :)
Can you do it the below way
```
<ReactTags
  inline
>
```
add the screenshot when inline true

```
<ReactTags
  inline={false}
>
```
Add the screenshot for the inline false

File: README.md [hunk 4]
```diff
@@ -365,6 +366,38 @@ The maxLength attribute added to the input. Specifies the maximum number of char
...
+
+![img](docs/inline-true.png)
+
+If inline is false
+
+```
+<div className="...">
+	{tagItems}
+</div>
+{tagInput}
```
Comment (ad1992) [inline README.md]: this can also be removed.

File: README.md (hunk 2 above)
Comment (hoantran-it) [reply README.md]: Hi @ad1992 can I use your screenshot, i think it's suitable :)

Comment (ad1992) [reply README.md]: sure 👍 @hoantran-it

Comment (hoantran-it) [reply README.md]: @ad1992 Thanks! Updated, please help to take a look :)

**Comment by ad1992** [comment]
looks good @hoantran-it :)

**Pull request merged**